assets/js/mock-data/batchEditData.js

```diff
@@ -0,0 +1,601 @@
+export const mockBatchEditData = [
+  {
+    accessionNumber: "Xample-b-25",
+    collection: null,
+    descriptiveMetadata: {
+      caption: [],
+      identifier: [],
+      title: "Sample Title 1",
+    },
+    id: "0239eeb9-1763-4c78-8948-853365e487dc",
+    representativeImage: null,
+  },
+  {
+    accessionNumber: "Xample-b-39",
+    collection: null,
+    descriptiveMetadata: {
+      caption: [],
+      identifier: [],
+      title: "Sample Title 2",
+    },
+    id: "0335b57f-33ad-4c85-bfd9-5260b2c183d6",
+    representativeImage:
+      "https://devbox.library.northwestern.edu:8183/iiif/2/ecb6ecbb-1c1a-4ee1-8697-6029b361d4ba",
```

Comment:
@katdivyareddy10 These are still pointing to local references, so they'll appear as broken.   Let's remove all references, and actually trim this file down to just 10 items to keep it manageable.  It's just test data after all which will go away soon.


![image](https://user-images.githubusercontent.com/3020266/87069878-9205ac80-c1dd-11ea-89bf-dda02c5c85f0.png)

Reply:
Just force-pushed the changes. I forgot those images were local to me. The links inside anchor tag must also be broken but they are set-up.